File: javascript/christmasTree.js
function treeBody() {
  for (let i = 1; i <= 6; i++) { 
    let consoleRow = ""; 
    for(let j = 6 - i; j > 0; j--){
      consoleRow += " ";
    } 
    for(let j = i *2 -1; j > 0; j--){
      consoleRow += "^";
    }
    console.log(consoleRow);
  }
};
treeBody()

function treeBottom(){
  for (let i = 0; i < 2; i++){
    whitespace = " "
    material = "*"
    console.log((whitespace.repeat(4)) + (material.repeat(3)) + (whitespace.repeat(4)))
  }
}; 
treeBottom()

File: python/christmas_tree.py
def tree_body():
    for i in range(6):
        for j in range( 6 - i):
            print(' ', end='')
        for k in range(i * 2 + 1):
            print('^',end='')
        print()
        
tree_body()

def tree_bottom():
    for i in range(0, 2):
        print(((' ')*5) + (('*')*3) + ((' ')*5))

tree_bottom()